packages/cardpay-subgraph-extraction/config/safe_owner.yaml

```diff
@@ -0,0 +1,32 @@
+base: &base
+  name: safe_owner
+  tables:
+    safe_owner:
+      column_mappings:
+        created_at:
```

Comment:
Cool. Had to check this `created_at` field was necessary since most `ownership_changed_at` was upon creation. Guess we wud want to be able to distinguish the changes after creation.

Reply:
Yeah technically we could leave it out of the mappings but easy to add and leave it in.